src/ensembl/src/shared/badged-button/BadgedButton.scss

```diff
@@ -0,0 +1,20 @@
+@import 'src/styles/common';
+
+.badgedButton {
+  position: relative;
+  display: inline-flex;
```

Comment:
Why do we want flex here?

Reply:
Not sure why I tried to use flex instead of `inline-block`. I've updated it.